src/reaper_osara.cpp

```diff
@@ -3713,11 +3713,18 @@ void moveToItem(int direction, bool clearSelection=true, bool select=true) {
 		if (take) {
 			s << " " << GetTakeName(take);
 		}
-		int takeCount = CountTakes(item);
-		if (takeCount > 1) {
+		int totalTakes = CountTakes(item);
```

Comment:
This is used twice, so let's split this into a utility function in the utility area at the top of this file: countNonEmptyTakes or similar.